example/src/tests.ts

```diff
@@ -620,3 +620,11 @@ test("canManagePreferences", async () => {
   return true
 });
 
+test("is address in the XMTP network", async () => {
+  const alix = await XMTP.Client.createRandom({ env: "local" });
+
+  const isAddressAvailable = await XMTP.publicCanMessage(alix.address, 'local');
+
+  return isAddressAvailable
```

Comment:
Seems like you should also add a test that it returns false for addresses not on the network as well because you had a bug in the android code that would always return true that this didn't catch.